pinot-plugins/pinot-stream-ingestion/pinot-kafka-2.0/src/test/java/org/apache/pinot/plugin/stream/kafka20/KafkaPartitionLevelConsumerTest.java

```diff
@@ -54,11 +54,11 @@
   private static final String TEST_TOPIC_2 = "bar";
   private static final int NUM_MSG_PRODUCED_PER_PARTITION = 1000;
 
-  private static MiniKafkaCluster kafkaCluster;
-  private static String brokerAddress;
+  private MiniKafkaCluster kafkaCluster;
```

Comment:
Curious on how the changes in these files are related to the fix?

Reply:
I was seeing everytime the tests failed on this file, so just modify it to play around